XenModel/Messages.resx

```diff
@@ -6380,6 +6383,9 @@ Click Configure HA to alter the settings displayed below.</value>
   <data name="HOST_GONE" xml:space="preserve">
     <value>Could not find host in [XenCenter]'s cache.</value>
   </data>
+  <data name="HOST_HAS_PV_GUEST_WARNING" xml:space="preserve">
+    <value>If you are upgrading to [XenServer] 8.1.0 or higher, some PV OS templates will be removed.</value>

```

Comment:
Versions should not be hardcoded. Please use the variable BRANDING_XC_PRODUCT_8_1_VERSION (defined in `xenadmin.git/Branding/branding.sh`.

Reply:
We need 2 different messages, one for updating and one for upgrading. Also, the message should not describe the code change, but rather the consequences for the user. For example, something like:
`Your PV guests may not function properly after the upgrade/update because support for PV guests is dropped as of [XenServer] [BRANDING_XC_PRODUCT_8_1_VERSION].`

Reply:
Ah, I forgot to mention that the version the user is upgrading to can be found using the prepare_host_upgrade.py plugin. Please see the logic in HostMemoryPostUpgradeCheck.cs for details.